app/controllers/api/v1/groups_controller.rb

```diff
@@ -5,14 +5,56 @@ class Api::V1::GroupsController < Api::V1::BaseApiController
 
   # endpoint to obtain all groups
   def index
-    groups = @assessment.groups
+    show_members = ActiveModel::Type::Boolean.new.cast(params[:show_members]&.downcase) || false
```

Comment:
I thought the same at first but `downcase` is important in case the param is capitalized:

<img width="518" alt="Screenshot 2024-01-05 at 2 08 43 PM" src="https://github.com/autolab/Autolab/assets/30433769/eb4859cc-905f-41ee-b7ba-00d3380b178a">


Does `to_boolean` handle this, or should it handle this?

Reply:
Ah I see, in that case could you update the definition of `to_boolean` in `core_ext.rb` to `ActiveRecord::Type::Boolean.new.cast(self.downcase)`?